### ISSUE TITLE
can't AoT required namespaces (clojure.spec.alpha)

### ISSUE THREAD

**Issue by selfsame** [arcadia-unity/ArcadiaGodot#1]
***Could not locate clojure.spec.alpha.cljc.dll or clojure/spec/alpha.cljc on load path.***

```
OpenGL ES 3.0 Renderer: GeForce GTX 960/PCIe/SSE2
Mono: Logfile is: /home/selfsame/.local/share/godot/app_userdata/waiting/mono/mono_logs/2019_05_03 08.43.57 (14940).txt
Starting Arcadia..

Unhandled Exception:
System.TypeInitializationException: The type initializer for 'clojure.lang.RT' threw an exception. ---> clojure.lang.Compiler+AssemblyInitializationException: Error initializing clojure.core.server.clj, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeInitializationException: The type initializer for '__expr86221' threw an exception. ---> clojure.lang.Compiler+AssemblyInitializationException: Error initializing clojure.main.clj, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeInitializationException: The type initializer for '__expr73290' threw an exception. ---> System.IO.FileNotFoundException: Could not locate clojure.spec.alpha.cljc.dll or clojure/spec/alpha.cljc on load path.
  at clojure$core$load_libfn__80245__80249.invoke () [0x00074] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_lib__80256.invokeStatic (System.Object , System.Object , clojure.lang.ISeq ) [0x0045d] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_lib__80256.doInvoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.applyTo (clojure.lang.ISeq arglist) [0x000d2] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$apply__74241.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$apply__74241.invoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_libs__80281.invokeStatic (clojure.lang.ISeq ) [0x00619] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_libs__80281.doInvoke (System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.applyTo (clojure.lang.ISeq arglist) [0x0008c] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$apply__74241.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$apply__74241.invoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$require__80303.invokeStatic (clojure.lang.ISeq ) [0x00019] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$require__80303.doInvoke (System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.invoke (System.Object arg1) [0x00029] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$main$loading__66432__auto____73285__73288.invokeStatic () [0x0003f] in <c0efc75ad0594cd284af9e0f786723cc>:0 
  at clojure$main$loading__66432__auto____73285__73288.invoke () [0x00000] in <c0efc75ad0594cd284af9e0f786723cc>:0 
  at __Init__$clojure$main+__expr73290..cctor () [0x00005] in <c0efc75ad0594cd284af9e0f786723cc>:0 
   --- End of inner exception stack trace ---
  at (wrapper managed-to-native) System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor(intptr)
  at System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor (System.RuntimeTypeHandle type) [0x0002a] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at clojure.lang.Compiler.InitializeNamespace (System.String nsName, System.String[] names, System.Type[] types, clojure.lang.Compiler+StaticMetadataFlags[] metadataFlags, System.Type[] metadataTypes) [0x00087] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at __Init__$clojure$main.Initialize () [0x00033] in <c0efc75ad0594cd284af9e0f786723cc>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0003b] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 

   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00051] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at System.RuntimeType.InvokeMember (System.String name, System.Reflection.BindingFlags bindingFlags, System.Reflection.Binder binder, System.Object target, System.Object[] providedArgs, System.Reflection.ParameterModifier[] modifiers, System.Globalization.CultureInfo culture, System.String[] namedParams) [0x00680] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at System.Type.InvokeMember (System.String name, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object target, System.Object[] args) [0x00000] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at clojure.lang.Compiler.InvokeInitType (System.Reflection.Assembly assy, System.Type initType) [0x00010] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
   --- End of inner exception stack trace ---
  at clojure$core$load_libfn__80245__80249.invoke () [0x00074] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_lib__80256.invokeStatic (System.Object , System.Object , clojure.lang.ISeq ) [0x0045d] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_lib__80256.doInvoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.applyTo (clojure.lang.ISeq arglist) [0x000d2] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$apply__74241.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$apply__74241.invoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_libs__80281.invokeStatic (clojure.lang.ISeq ) [0x00619] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_libs__80281.doInvoke (System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.applyTo (clojure.lang.ISeq arglist) [0x0008c] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$apply__74241.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$apply__74241.invoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$require__80303.invokeStatic (clojure.lang.ISeq ) [0x00019] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$require__80303.doInvoke (System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.invoke (System.Object arg1, System.Object arg2, System.Object arg3) [0x00055] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$server$loading__79968__auto____86216__86219.invokeStatic () [0x00035] in <1d8b9519d64d4a9eba12a49b914cf479>:0 
  at clojure$core$server$loading__79968__auto____86216__86219.invoke () [0x00000] in <1d8b9519d64d4a9eba12a49b914cf479>:0 
  at __Init__$clojure$core$server+__expr86221..cctor () [0x00005] in <1d8b9519d64d4a9eba12a49b914cf479>:0 
   --- End of inner exception stack trace ---
  at (wrapper managed-to-native) System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor(intptr)
  at System.Runtime.CompilerServices.RuntimeHelpers.RunClassConstructor (System.RuntimeTypeHandle type) [0x0002a] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at clojure.lang.Compiler.InitializeNamespace (System.String nsName, System.String[] names, System.Type[] types, clojure.lang.Compiler+StaticMetadataFlags[] metadataFlags, System.
Type[] metadataTypes) [0x00087] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at __Init__$clojure$core$server.Initialize () [0x00033] in <1d8b9519d64d4a9eba12a49b914cf479>:0 
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0003b] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00051] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at System.RuntimeType.InvokeMember (System.String name, System.Reflection.BindingFlags bindingFlags, System.Reflection.Binder binder, System.Object target, System.Object[] providedArgs, System.Reflection.ParameterModifier[] modifiers, System.Globalization.CultureInfo culture, System.String[] namedParams) [0x00680] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at System.Type.InvokeMember (System.String name, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object target, System.Object[] args) [0x00000] in <52405d1838e24bc5b300f5ebeee9f7c4>:0 
  at clojure.lang.Compiler.InvokeInitType (System.Reflection.Assembly assy, System.Type initType) [0x00010] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
   --- End of inner exception stack trace ---
  at clojure$core$load_libfn__80245__80249.invoke () [0x00074] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_lib__80256.invokeStatic (System.Object , System.Object , clojure.lang.ISeq ) [0x0045d] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_lib__80256.doInvoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.applyTo (clojure.lang.ISeq arglist) [0x000d2] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$apply__74241.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$apply__74241.invoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_libs__80281.invokeStatic (clojure.lang.ISeq ) [0x00619] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$load_libs__80281.doInvoke (System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.applyTo (clojure.lang.ISeq arglist) [0x0008c] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$apply__74241.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$apply__74241.invoke (System.Object , System.Object , System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$require__80303.invokeStatic (clojure.lang.ISeq ) [0x00019] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure$core$require__80303.doInvoke (System.Object ) [0x00000] in <2c8eb2e0b500482da19b86cf3bfb2229>:0 
  at clojure.lang.RestFn.invoke (System.Object arg1) [0x00029] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.Var.invoke (System.Object arg1) [0x00010] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.RT.PostBootstrapInit () [0x000bd] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.RT.DoInit () [0x00011] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.RT..cctor () [0x01f3d] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
   --- End of inner exception stack trace ---
  at ArcadiaHook.Initialize () [0x00070] in <667f4df1bba64e29a726a8c87476141a>:0 
  at ArcadiaHook._EnterTree () [0x00012] in <667f4df1bba64e29a726a8c87476141a>:0 
```

**Comment by selfsame** [comment]
trying to compile `clojure.spec.alpha` gives:

```
clojure.lang.Compiler+CompilerException: System.Exception: #object[alpha$and_spec_impl$reify__16994__17005 0xdc0cbc69 "clojure.spec.alpha$and_spec_impl$reify__16994__17005"] is not a fn, expected predicate fn
  at clojure$spec$alpha$dt__16583.invokeStatic (System.Object , System.Object , System.Object , System.Object ) [0x0015e] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$dt__16583.invoke (System.Object , System.Object , System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$dt__16583.invokeStatic (System.Object , System.Object , System.Object ) [0x0000a] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$dt__16583.invoke (System.Object , System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invokeStatic (System.Object , System.Object ) [0x00250] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invokeStatic (System.Object , System.Object ) [0x005d2] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$derivfn__17452__17456.invoke (System.Object ) [0x0000a] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$mapfn__21637__21641.invoke () [0x001ae] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure.lang.LazySeq.sval () [0x00008] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.sval()
  at clojure.lang.LazySeq.seq () [0x00000] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.seq()
  at clojure.lang.RT.seq (System.Object coll) [0x00016] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$seq__19093__19096.invokeStatic (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$seq__19093__19096.invoke (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$mapfn__21653__21657.invoke () [0x0000a] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure.lang.LazySeq.sval () [0x00008] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.sval()
  at clojure.lang.LazySeq.seq () [0x00000] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.seq()
  at clojure.lang.RT.seq (System.Object coll) [0x00016] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$seq__19093__19096.invokeStatic (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$seq__19093__19096.invoke (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$filterfn__21739__21743.invoke () [0x0000a] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure.lang.LazySeq.sval () [0x00008] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.sval()
  at clojure.lang.LazySeq.seq () [0x00000] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.seq()
  at clojure.lang.RT.seq (System.Object coll) [0x00016] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$seq__19093__19096.invokeStatic (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$seq__19093__19096.invoke (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$mapfn__21637__21641.invoke () [0x0000a] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure.lang.LazySeq.sval () [0x00008] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.sval()
  at clojure.lang.LazySeq.seq () [0x00000] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at (wrapper synchronized) clojure.lang.LazySeq.seq()
  at clojure.lang.RT.seq (System.Object coll) [0x00016] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure$core$seq__19093__19096.invokeStatic (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$core$seq__19093__19096.invoke (System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$filter_alt__17297.invokeStatic (System.Object , System.Object , System.Object , System.Object ) [0x0016e] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$filter_alt__17297.invoke (System.Object , System.Object , System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$alt_STAR___17312.invokeStatic (System.Object , System.Object , System.Object ) [0x0001c] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$alt_STAR___17312.invoke (System.Object , System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invokeStatic (System.Object , System.Object ) [0x0030f] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invokeStatic (System.Object , System.Object ) [0x005d2] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invokeStatic (System.Object , System.Object ) [0x006e2] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$deriv__17459.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$re_conform__17554.invokeStatic (System.Object , System.Object ) [0x00117] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$re_conform__17554.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure.spec.alpha$regex_spec_impl$reify__17575__17579.conform_STAR_ (System.Object ) [0x00085] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$conform__15881.invokeStatic (System.Object , System.Object ) [0x0005c] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$conform__15881.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$macroexpand_check__16544.invokeStatic (System.Object , System.Object ) [0x00091] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure$spec$alpha$macroexpand_check__16544.invoke (System.Object , System.Object ) [0x00000] in <372fd84fa83c4ac19e6ca7ef0c5aecf7>:0 
  at clojure.lang.AFn.ApplyToHelper (clojure.lang.IFn fn, clojure.lang.ISeq argList) [0x000a6] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.AFn.applyTo (clojure.lang.ISeq arglist) [0x0000b] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.Var.applyTo (clojure.lang.ISeq arglist) [0x00006] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.Compiler.CheckSpecs (clojure.lang.Var v, clojure.lang.ISeq form) [0x00028] in <fb1204dbf0b848beaab1559f81f1b22c>:0 , compiling: (NO_SOURCE_PATH:0:0)
  at clojure.lang.CljCompiler.Ast.InvokeExpr.Parse (clojure.lang.CljCompiler.Ast.ParserContext pcon, clojure.lang.ISeq form) [0x0001b] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.CljCompiler.Ast.BodyExpr+Parser.Parse (clojure.lang.CljCompiler.Ast.ParserContext pcon, System.Object frms) [0x0005c] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.CljCompiler.Ast.BodyExpr+Parser.Parse (clojure.lang.CljCompiler.Ast.ParserContext pcon, System.Object frms) [0x00048] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.CljCompiler.Ast.TryExpr+Parser.Parse (clojure.lang.CljCompiler.Ast.ParserContext pcon, System.Object frm) [0x0024a] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.CljCompiler.Ast.IfExpr+Parser.Parse (clojure.lang.CljCompiler.Ast.ParserContext pcon, System.Object frm) [0x00054] in <fb1204dbf0b848beaab1559f81f1b22c>:0 
  at clojure.lang.CljCompiler.Ast.BodyExpr+Parser.Parse (clojure.lang.CljCompiler.Ast.ParserContext pcon, System.Object f
```

**Comment by selfsame** [comment]
closed with 2dba97c234921787081cada716233a4a431e90cb
